### PR TITLE
add the functionality to track consumer group offsets

### PR DESCRIPTION
**Pull request by sli-cs**
https://app.clubhouse.io/callstatsio/story/8326/prometheus-kafka-probe-improvement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,4 +30,6 @@ Application reads configuration from following environment variables
 | Name       | Description                         |
 | ---------- | ----------------------------------- |
 | PORT       | HTTP port to listen to              |
-| ZOOKEEPERS | Comma separated list of Zookeepers  |
+| KAFKAHOSTS | Comma separated list of kafka broker hosts  |
+| TOPICS     | Comma separated list of topics to scrape |
+| CONSUMERGROUPS | a json string specifying the topic and the list of consumer groups |
```
Comment (eljasala) [inline README.md]: Add a bit more detailed explanation about the schema (e.g. map of topic name to array of consumer groups or something similar).

File: helper.py [hunk 2]
```diff
@@ -2,11 +2,12 @@
 
 import logging
 import pykafka
+
 LOGGER = logging.getLogger(__name__)
 
 class KafkaHelper:
-    def __init__(self, zookeeper_host, broker_version='0.9.0'):
-        self.client = pykafka.KafkaClient(zookeeper_hosts=zookeeper_host, broker_version=broker_version)
+    def __init__(self, kafka_hosts, broker_version='0.9.0'):
```
Comment (eljasala) [inline helper.py]: Does `broker_version` matter?

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-kazoo==2.4.0
-pykafka==2.6.0
+kazoo==2.5.0
```
Comment (eljasala) [inline requirements.txt]: Probably not needed anymore

**Review by eljasala — APPROVED** [review]

File: helper.py (hunk 2 above)
Comment (sli-cs) [reply helper.py]: Not really needed. I'll remove it.

**Pull request merged**